### DISCUSSION TITLE
defaultValue vs. initialValue

### DISCUSSION THREAD

**Discussion by marcomuser** [edmundhung/conform#417] [Q&A] [answered]
I have a specific usecase where I want to set an initialValue to a field but only if the user didn't submit a value for it yet. How would you accomplish this behavior with conform v1.0? I had hoped it would be possible to set a value to a field that only applies if the defaultValues coming from the BE did not include a value for this field yet.

**Comment by edmundhung** [comment, marked as answer]
Can you elaborate more what you are trying to achieve here? Maybe you are looking for [`.default`](https://zod.dev/?id=default) from zod?

**Reply by marcomuser** [reply]
Yeah you are totally right.  Was really just looking for .default. Not sure what my thought  process was at the time of writing 😅 anyways, thanks for clarifying!
